### ISSUE TITLE
Cannot install Mac 05 version

### ISSUE THREAD

**Issue by johnbourne** [armory3d/armory#1138]
*(drag & drop zipped .blend file here)*
Could you check the Armory_05 version for the Mac.  I've downloaded it several times and it always says when run that the file is corrupted.  
In contrast the Armory_06 version for the Mac downloads and runs fine.  I'd like to use the 05 version for a while, but will see how far I get the the 06 version for 2.8.

**Comment by BlackGoku36** [comment]
https://armory3d.org/manual/#/getting_started/setup?id=macos

**Comment by luboslenco** [comment]
Hopefully running from terminal helped!

**Comment by johnbourne** [comment]
Yes, but I should have mentioned that this works even better:

sudo spctl --master-disable

p.s.  all is well so far with 06 beta experiments, but I am finding
difficulty with lighting.  Sometimes object disappear unless lighting is
recreated.  I've seen some comments that 06 is almost out and "glaring"
problems fixed.  I'm looking forward to it...

John R. Bourne

**Comment by luboslenco** [comment]
Objects disappearing sounds bad, if you manage to run into it again and then share the blend file here that would be super cool. :)

**Comment by johnbourne** [comment]
Sure, here is the problem I ran into —
All I had originally was one cube and was experimenting with lighting.  I
saved the file, reopened and the cube disappeared.  (well the top was
left).  I add the cube to the right and that could be then seen.  It was
almost as if the lighting influenced whether the cube could be seen.  In
fact, the first time I had to move the lighting to see the cube at all.  I
used a HDRI for lighting and the spot light for additional light.  Rendered
with Krom - seemed to be the same with the browser.  Used the player
controller and camera controller nodes.

File attached...

John
